tests/schema/mutations/updateSection.test.js

```diff
@@ -12,6 +12,38 @@ describe("updateSection", () => {
     }
   `;
 
+  const createSectionIntroduction = `
+  mutation CreateIntroduction($input: CreateIntroductionInput!) {
+    createIntroduction(input: $input) {
+      title
+      content
+    }
+  }`;
+
+  const updateSectionIntroduction = `
+  mutation UpdateIntroduction($input: UpdateIntroductionInput!) {
+    updateIntroduction(input: $input) {
+      title
+      content
+    }
+  }`;
+
+  const deleteSectionIntroduction = `
+  mutation DeleteIntroduction($input: DeleteIntroductionInput!) {
+    deleteIntroduction(input: $input) {
+      title
+      content
+    }
+  }`;
+
+  const undeleteSectionIntroduction = `
+  mutation UndeleteIntroduction($input: UndeleteIntroductionInput!) {
+    undeleteIntroduction(input: $input) {
+      title
+      content
+    }
+  }`;
```

Comment:
Small nitpick but could you import these mutations from the `utils/graphql` file, saves them from being written out twice?